src/theme.js

```diff
@@ -99,9 +108,27 @@ const theme = createMuiTheme({
       right: 24,
     },
     greenIcon: {
-      backgroundColor: SUCCESS.MAIN,
+      backgroundColor: success.main,
+      '&:hover': {
+        backgroundColor: success.dark,
+      },
+      '& svg': {
+        backgroundColor: 'transparent',
+      },
+    },
+    yellowIcon: {
```

Comment:
I'd like to see these named by intent rather than color. So `successIcon`, `warningIcon`, and `errorIcon`, so they align with the palettes.